### ISSUE TITLE
[Multiple File Upload] Some text reads twice when walking through question descriptions/titles.

### ISSUE THREAD

**Issue by devin-civi** [civiform/civiform#8722]
I suspect this may be present with the single file upload too since it has to do with the actual question content, but it's worth fixing regardless.

Melanie included a video: https://civiform.slack.com/archives/C07KXU02B70/p1727284843249889
